Pipfile

```diff
@@ -4,25 +4,26 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+azure-identity = "~=1.17"
 better-abc = ">=0.0.3"
 marshmallow-dataclass = "~=8.5"
 numpy = ">=1.18"
 oic = "~=1.7.0"
 pandas = ">=1.4"
-requests = "~=2.14"
-typeguard = ">=2.13"
 pbr = "~=6.0"
+pyarrow = "~=17.0.0"
```

Comment:
Added pyarrow to work with parquet files.
Other changes are because of sorting.